ci-operator/step-registry/cucushift/installer/rehearse/alibabacloud/ipi/cucushift-installer-rehearse-alibabacloud-ipi-workflow.yaml

```diff
@@ -1,17 +1,9 @@
 workflow:
   as: cucushift-installer-rehearse-alibabacloud-ipi
-  steps: 
+  steps:
     pre:
-    - chain: cucushift-installer-rehearse-alibabacloud-ipi-provision
+      - chain: cucushift-installer-rehearse-alibabacloud-ipi-provision
+      - ref: cucushift-installer-reportportal-marker
```

Comment:
I agree that the format/indention is much better than the previous one.  The issue here is that it makes the review not easy for this PR.  Is it fine to do the format change in another PR ?

Reply:
Maybe we can workaround it by using `sed` as `find ci-operator/step-registry/cucushift/installer/rehearse/ -name "*-workflow.yaml" -type f -exec sed -i -E 's/( +post:)/    - ref: cucushift-installer-reportportal-marker\n\1/' {} \;`

Reply:
The format change is not a issue. The issue is that the format change may hide unexpected non-format changes.